trade-finance-manager-ui/server/schemas/with-entra-id-user-schema.tests.ts

```diff
@@ -0,0 +1,149 @@
+import { ZodObject } from 'zod';
+import { withSchemaTests } from '../../test-helpers';
+import { EntraIdUser } from '../types/entra-id-user';
+import { ENTRA_ID_USER_GROUP } from '../constants/entra-id-user';
+
+type TestCasesParams = {
+  getValidObjectWithUpdatedEntraIdUserParams: (entraIdUser: unknown) => unknown;
+};
+
+type withEntraIdUserSchemaTestsParams = {
+  schema: ZodObject<any>;
```

Comment:
Can we replace `any` with a real type?

Reply:
This is a real type -- I've kept this rather wide to allow for compatibility with zod objects, but all ZodObjects can have `.safeParse` called on it (hence no type error!)

Reply:
I've been more specific here...update inbound